pkg/fleetautoscalers/fleetautoscalers.go

```diff
@@ -321,8 +321,20 @@ func applyCounterOrListPolicy(c *autoscalingv1.CounterPolicy, l *autoscalingv1.L
 		}
 		// The desired TOTAL capacity based on the Aggregated Allocated Counts (see applyBufferPolicy for explanation)
 		desiredCapacity := int64(math.Ceil(float64(aggAllocatedCount*100) / float64(100-bufferPercent)))
+
+		// Ensures desiredCapacity is at least 1
+		if desiredCapacity < 1 {
```

Comment:
Since you check `if buffer < 1` you don't need to check desired capacity.

Reply:
I've eliminated the desiredCapacity check